internal/pkg/show/status/output.go

```diff
@@ -0,0 +1,79 @@
+package status
+
+import (
+	"bytes"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Output struct {
+	Name    string                 `yaml:"Name"`
+	Plugin  string                 `yaml:"Plugin"`
+	Drifted bool                   `yaml:"Drifted"`
+	Options map[string]interface{} `yaml:"Options"`
+	Status  *Status                `yaml:"Status"`
+}
+
+type Status struct {
+	InnerStatus map[string]interface{} `yaml:",inline,omitempty"`
```

Comment:
The comment below is very detailed, but here the naming of the variables can be improved.

Reply:
How about InlineStatus?